### ISSUE TITLE
browse_website: Config needs to be passed to nested summarize calls

### ISSUE THREAD

**Issue by codiak** [Significant-Gravitas/Auto-GPT#4949]
After running into this error on a clean build of the latest release:
```
Command browse_website returned: Error: 'NoneType' object has no attribute 'fast_llm_model'
```

I traced it back to these lines in `text.py`:
https://github.com/Significant-Gravitas/Auto-GPT/blob/80151dd6b2870dc6f91aafadb3a451f2e440a3ba/autogpt/processing/text.py#L134C1-L139

And found that "config" still needed to be passed in to additional calls to summarize_text():
```
        summary, _ = summarize_text(chunk, config, instruction)
        summaries.append(summary)

    logger.info(f"Summarized {len(chunks)} chunks")

    summary, _ = summarize_text("\n\n".join(summaries), config)
```

**Comment by GECORegulatory** [comment]
@codiak Please try my workaround in #4973 and let me know if that works. If not, feel free to hop in the discord. 

Thanks,
-GECO

**Comment by collijk** [comment]
Fixed in #4863
